Dockerfile.dev

```diff
@@ -16,6 +16,7 @@ WORKDIR /app
 # ... really speeds up building new containers
 COPY requirements.txt /app/
 RUN pip install -r requirements.txt
+RUN pip install tox
```

Comment:
We recently enabled hash verification for everything in requirements.txt. Is there a way to do that for tox, as well?